staging/src/k8s.io/apiserver/pkg/storage/interfaces.go

```diff
@@ -236,6 +236,11 @@ type Interface interface {
 
 	// Count returns number of different entries under the key (generally being path prefix).
 	Count(key string) (int64, error)
+
+	// RequestBookmark requests a bookmark to be sent on opened watches.
+	// Note: only watches with matching context metadata are notified.
```

Comment:
Could you clarify what is "matching context metadata"? Like is it some specific key in the context, or hash of all data or what?

Reply:
Done https://github.com/kubernetes/kubernetes/blob/9325a57125e8502941d1b0c7379c4bb80a678d5c/vendor/go.etcd.io/etcd/client/v3/watch.go#L1037-L1042